Xamarin.Forms.Core/SwipeView.cs

```diff
@@ -64,6 +69,12 @@ public SwipeItems BottomItems
 			set { SetValue(BottomItemsProperty, value); }
 		}
 
+		public bool AutoClose
```

Comment:
I have thought different ideas. This option is the simplest one, my question is, is it easy to understand?.